CHANGELOG.md

```diff
@@ -127,6 +127,8 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 * `jj resolve` now accepts new built-in merge tools `:ours` and `:theirs`.
   These merge tools accept side #1 and side #2 of the conflict respectively.
 
+* `jj config edit` will now rollback to previous version if an error has been induced in the new config.
```

Comment:
Just reminder. You'll need to move the changelog entry.

Reply:
Into fixed bugs ? Already moved it into New features

Reply:
Into "[Unreleased]"

Reply:
ooh ok my bad. I'll fix and merge asap